Components/Model/Customer.php

```diff
@@ -36,6 +36,7 @@
 
```

Comment:
Extend https://github.com/Nosto/nosto-php-sdk/blob/develop/src/Object/Customer.php

Reply:
Maybe we need in order to use the loadData method. I'm not aware if we can load those attributes with the SDK.